CHANGELOG.md

```diff
@@ -0,0 +1,477 @@
+-   [JENKINS-33623](https://issues.jenkins-ci.org/browse/JENKINS-33623): Proxy configuration not correctly handling wildcards.
+-   Added more logging about webhooks in pull requests.
+
+## Version 1.7 (May 13, 2016)
+
+-   [JENKINS-34727](https://issues.jenkins-ci.org/browse/JENKINS-34727): WebHook events are not always successfully 
+    triggering Jenkins pipeline
+-   [JENKINS-34776](https://issues.jenkins-ci.org/browse/JENKINS-34776): Jobs are removed if the remote is unavailable
+-   Added extra log messages from WebHook processing
+
+## Version 1.6 (Apr 27, 2016)
+
+-   [JENKINS-34410](https://issues.jenkins-ci.org/browse/JENKINS-34410):
+    Improve the search procedure of `SCRIPT_FILE`, when you work with
+    Pipeline Multibranch projects is `Jenkinsfile`.
+-   [JENKINS-34237](https://issues.jenkins-ci.org/browse/JENKINS-34237):
+    GitHub Organizations and GitHub User Accounts are searched using
+    insensitive case
+-   [JENKINS-33318](https://issues.jenkins-ci.org/browse/JENKINS-33318):
+    GitHub Enterprise server validation with private mode enabled
+-   [JENKINS-33305](https://issues.jenkins-ci.org/browse/JENKINS-33318):
+    Branch name filters at GitHub Organization folder level
+
+## Version 1.5 (Apr 11, 2016)
+
+-   [JENKINS-33808](https://issues.jenkins-ci.org/browse/JENKINS-33808):
+    Support for Item categorization. More information about this new
+    feature in core here
+    [JENKINS-31162](https://issues.jenkins-ci.org/browse/JENKINS-31162):
+-   [JENKINS-33306](https://issues.jenkins-ci.org/browse/JENKINS-33306):
+    Improve the error handling when repo:status privilege missing
+-   [JENKINS-34047](https://issues.jenkins-ci.org/browse/JENKINS-34047):
+    Sort GitHub repositories in case insensitive
+-   [JENKINS-33815](https://issues.jenkins-ci.org/browse/JENKINS-33815):
+    Validation for GitHub Organizations and GitHub User Accounts
+
+## Version 1.4 (Mar 14, 2016)
+
+-   [JENKINS-33256](https://issues.jenkins-ci.org/browse/JENKINS-33256): The ability to build pull requests to public
+    repositories was restored. In the case of multibranch Pipeline projects, for submitters who are not collaborators 
+    on the repository, the PR will be built, but using the `Jenkinsfile` from the base branch.
+-   Implemented some caching of GitHub API calls to improve performance.
+-   [JENKINS-33309](https://issues.jenkins-ci.org/browse/JENKINS-33309): Implementing an API to list metadata about a 
+    pull request; also available as environment variables during a build, and from the REST API for a job.
+-   [JENKINS-33183](https://issues.jenkins-ci.org/browse/JENKINS-33183): Fix usage of anonymous scan credentials. (Still
+    inadvisable due to GitHub rate limits.)
+-   Need an *Add* button under *Checkout credentials*.
+-   Sorting repositories in the GitHub branch source configuration screen.
+-   Miscellaneous UX improvements, including error messages.
+
+## Version 1.3 (Feb 26, 2015)
```

Comment:
Just noticed that these two jump from 2016 to 2015, which is wrong. I'm going to correct that now.

Reference: [the releases page here on GitHub](https://github.com/jenkinsci/github-branch-source-plugin/releases?after=github-branch-source-1.8.1).

<img width="664" alt="image" src="https://user-images.githubusercontent.com/21689198/72856538-d1b8f600-3c88-11ea-927a-b6ae3f07bff8.png">